cluster/photon-controller/config-common.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##########################################################
+#
+# Parameters needed for interacting with Photon Controller
+#
+##########################################################
+
+# Pre-created tenant for Kubernetes to use
+PHOTON_TENANT=kube-tenant
+
+# Pre-created project in PHOTON_TENANT for Kubernetes to use
+PHOTON_PROJECT=kube-project
+
+# Pre-created VM flavor for Kubernetes master to use
+# Can be same as master
+# We recommend at least 1GB of memory
+PHOTON_MASTER_FLAVOR=kube-vm
+
+# Pre-created VM flavor for Kubernetes node to use
+# Can be same as master
+# We recommend at least 2GB of memory
+PHOTON_NODE_FLAVOR=kube-vm
+
+# Pre-created disk flavor for Kubernetes to use
+PHOTON_DISK_FLAVOR=kube-disk
+
+# Pre-created Debian 8 image with kube user uploaded to Photon Controller
+PHOTON_IMAGE_ID=398e6dc2-d484-4fc8-a9a0-00b1fb4d1bf1
```

Comment:
Why is the image Id being hard-coded here? The image-Id will be different in every environment. Or is this more like a placeholder?